src/util/decrypt.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
```

Comment:
This strikes me as risky duplication - if we were to ever need to update enterprise server's encryption/decryption algorithms, we'd have to remember that there are 2 places in which that method is defined (formio-server and OSS formio). Could a better solution here be to encapsulate any decryption logic entirely from the formio-server? One method you might look into is by altering the "exporter" instance to include some sort of decryption subroutine that merely closes over the formio-server decryption fn. Your thoughts?

Reply:
We will need to implement a new hook that performs this decryption so we are not duplicating code here.